components/collector/tests/source_collectors/robot_framework/test_source_version.py

```diff
@@ -9,8 +9,12 @@ class RobotFrameworkSourceVersion(RobotFrameworkTestCase):
     METRIC_TYPE = "source_version"
     METRIC_ADDITION = "min"
 
-    async def test_source_version(self):
+    async def test_source_version3(self):
         """Test that the Robot Framework version is returned."""
-        xml = """<?xml version="1.0"?><robot generator="Robot 3.1.1 (Python 3.7.0 on darwin)"/>"""
-        response = await self.collect(get_request_text=xml)
-        self.assert_measurement(response, value="3.1.1")
+        response = await self.collect(get_request_text=self.ROBOT_FRAMEWORK_XML_V3)
+        self.assert_measurement(response, value="3.2.2")
+
+    async def test_source_version4(self):
```

Comment:
aren't these two tests for exactly the same code / exactly the same path in taken in the decision tree?